djangocms_blog/cms_wizards.py

```diff
@@ -97,25 +97,30 @@ class PostWizard(Wizard):
     pass
 
 
-for config in BlogConfig.objects.all().order_by("namespace"):
-    seed = slugify("{}.{}".format(config.app_title, config.namespace))
-    new_wizard = type(str(seed), (PostWizard,), {})
-    new_form = type("{}Form".format(seed), (PostWizardForm,), {"default_appconfig": config.pk})
-    post_wizard = new_wizard(
-        title=_("New {0}").format(config.object_name),
-        weight=200,
-        form=new_form,
-        model=Post,
-        description=_("Create a new {0} in {1}").format(config.object_name, config.app_title),
-    )
-    try:
-        wizard_pool.register(post_wizard)
-    except AlreadyRegisteredException:  # pragma: no cover
-        if settings.DEBUG:
-            raise
-        else:
-            warnings.warn(
-                "Wizard {} cannot be registered. Please make sure that "
-                "BlogConfig.namespace {} and BlogConfig.app_title {} are"
-                "unique together".format(seed, config.namespace, config.app_title)
-            )
+try:
```

Comment:
This needs moving to the cms_config to work using the new app registration system which replaces pools: https://github.com/django-cms/django-cms/blob/28d3fa54cb8795fc2276ada94e07f45fa2fe1def/cms/cms_config.py#L18